lib/redis_memo/memoize_query/invalidation.rb

```diff
@@ -258,9 +253,12 @@ def self.select_by_new_ids(model_class, target_id)
     end
   end
 
-  def self.bulk_operations_invalidation_limit
-    ENV['REDIS_MEMO_BULK_OPERATIONS_INVALIDATION_LIMIT']&.to_i ||
-      RedisMemo::DefaultOptions.bulk_operations_invalidation_limit ||
-      10000
+  def self.select_by_conflict_target_relation(model_class, relation)
+    RedisMemo::Tracer.trace(
```

Comment:
I think having the invalidation limit here is still helpful, what's the reasoning behind removing it? (Also might be helpful for invalidating records in select_by_new_ids)

Reply:
Checking for `bulk_operations_invalidation_limit` would send another database query.

Records returned from `select_by_new_ids` and `select_by_conflict_target_relation` should be <= the number of records we're upserting anyway, so I don't think it's worth checking anymore.